gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/LoadJobConfiguration.java

```diff
@@ -0,0 +1,389 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.bigquery;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.api.services.bigquery.model.JobConfigurationLoad;
+import com.google.common.base.MoreObjects.ToStringHelper;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Google BigQuery load job configuration. A load job loads data from one of several formats into a
+ * table. Data is provided as URIs that point to objects in Google Cloud Storage. Load job
+ * configurations have {@link JobConfiguration.Type#LOAD} type.
+ */
+public class LoadJobConfiguration extends JobConfiguration implements LoadConfiguration {
+
+  private static final long serialVersionUID = -2673554846792429829L;
+
+  private final List<String> sourceUris;
+  protected final TableId destinationTable;
```

Comment:
Could these be private/package protected, or is it necessary that they're protected?